backends/p4tools/testgen/lib/tf.h

```diff
@@ -14,7 +16,7 @@ namespace P4Tools {
 namespace P4Testgen {
 
 /// THe default base class for the various test frameworks (TF). Every test framework has a test
-/// name and a seed associated with it.
+/// name and a seed associated with it. Also contains a variety of common utility functions.
```

Comment:
Do all test frameworks support `ActionProfiles` and `ActionSelectors` - do we want to move it to common tf?

Reply:
I think it is shared between enough back ends that we can add it to common.